cluster/addons/fluentd-elasticsearch/es-image/elasticsearch_logging_discovery.go

```diff
@@ -45,12 +46,20 @@ func main() {
 	if err != nil {
 		glog.Fatalf("Failed to make client: %v", err)
 	}
+	namespace := api.NamespaceSystem
+	envNamespace := os.Getenv("NAMESPACE")
+	if envNamespace != "" {
+		if _, err = c.Namespaces().Get(envNamespace); err != nil {
```

Comment:
sorry for the nit-picky comments, but this should be `if _, err := ...` instead of `if _, err = ...`. Also you should include the error in the `glog.Fatalf` line below so that we know what error caused the addon to exit.